src/shared/components/views/MarkdownRenderer.tsx

```diff
@@ -67,6 +67,11 @@ export const MarkdownRenderer: FC<Props> = ({
 
   // load images locally to your heart's content. caveat emptor
   return (
-    <ReactMarkdown source={text} className={className} escapeHtml={false} />
+    <ReactMarkdown
+      source={text}
+      className={className}
+      renderers={{...cloudRenderers}}
```

Comment:
```suggestion
      renderers={cloudRenderers}
```

Reply:
Thanks for the suggestion! I tried this out and it works fine. The component doesn't ever get more than one "renderer" in OSS mode.